boards/px4/fmu-v5/nuttx-config/include/board_dma_map.h

```diff
@@ -69,17 +69,17 @@
 | Channel 7  | -                | TIM8_UP          | TIM8_CH1_2       | TIM8_CH2_2       | TIM8_CH3_2       | SPI5_RX_2        | SPI5_TX_2        | TIM8_CH4         |
 |            |                  |                  |                  |                  |                  |                  |                  |   TIM8_TRIG      |
 |            |                  |                  |                  |                  |                  |                  |                  |   TIM8_COM       |
-| Channel 8  | DSFDM1_FLT0      | DSFDM1_FLT1      | DSFDM1_FLT2      | DSFDM1_FLT3      | DSFDM1_FLT0      | DSFDM1_FLT1      | DSFDM1_FLT2      | DSFDM1_FLT3      |
-| Channel 9  | JPEG_IN          | JPEG_OUT         | SPI4_TX          | JPEG_IN          | JPEG_OUT         | SPI5_RX          | -                | -                |
-| Channel 10 | SAI1_B           | SAI2_B           | SAI2_A           | -                | -                | -                | SAI1_A           | -                |
-| Channel 11 | SDMMC2           | -                | QUADSPI          | -                | -                | SDMMC2           | -                | -                |
+| Channel 8  | DFSDM1_FLT0_1    | DFSDM1_FLT1_1    | DFSDM1_FLT2_1    | DFSDM1_FLT3_1    | DFSDM1_FLT0_2    | DFSDM1_FLT1_2    | DFSDM1_FLT2_2    | DFSDM1_FLT3_2    |
+| Channel 9  | JPEG_IN_1        | JPEG_OUT         | SPI4_TX_3        | JPEG_IN_2        | JPEG_OUT_2       | SPI5_RX_3        | -                | -                |
+| Channel 10 | SAI1_B_3         | SAI2_B_1         | SAI2_A_1         | -                | -                | -                | SAI1_A_3         | -                |
+| Channel 11 | SDMMC2_1         | -                | QUADSPI_1        | -                | -                | SDMMC2_2         | -                | -                |
 |            |                  |                  |                  |                  |                  |                  |                  |                  |
-| Usage      | SPI1_RX_1        |                  | USART6_RX_2      | SPI1_TX_1        |                  | TIM1_UP          | SDMMC1_2         | USART6_TX_2      |
+| Usage      | SPI1_RX_1        |                  | USART6_RX_2      | SPI1_TX_1        |                  | TIM1_UP          | SDMMC1_2         |                  |
  */
 
 // DMA1 Channel/Stream Selections
 //--------------------------------------------//---------------------------//----------------
-//      DMAMAP_UART8_TX                       // DMA1, Stream 0, Channel 5    (PX4IO TX)
```

Comment:
The px4io usage is a bit weird because it's handled outside of NuttX.

https://github.com/PX4/Firmware/blob/224be8ba2473d334466316a9dee1bdaf332ca90d/boards/px4/fmu-v5/src/board_config.h#L60-L61

Reply:
ok, fixed on the table, but what about this? it that correct?
https://github.com/PX4/Firmware/blob/224be8ba2473d334466316a9dee1bdaf332ca90d/boards/px4/fmu-v5/nuttx-config/nsh/defconfig#L216

Reply:
Yes, and for clarity we might as well have `CONFIG_UART8_TXDMA=y` set as well.

Reply:
@dagar  ok, done.
Another look?